14_k8s_pod_volume_27-sep-2022/README.md

```diff
@@ -1,4 +1,4 @@
-## Demo on: Container orchestration, Kubernetes basics, part 2, 27 Sep 2022
+## Container orchestration Kubernetes basics: containers, pods, ephemeral volumes ##
 
```

Comment:
Specify the Linux distribution that is used to run all commands

Add Prerequisite section